geode-core/src/main/java/org/apache/geode/internal/cache/partitioned/PartitionMessage.java

```diff
@@ -333,15 +334,18 @@ public void process(final ClusterDistributionManager dm) {
           } else if (tx.isInProgress()) {
             sendReply = operateOnPartitionedRegion(dm, pr, startTime);
             tx.updateProxyServer(this.getSender());
+          } else {
+            throw new TransactionException("transactional operation elided because transaction {"
```

Comment:
Probably can add a comment that this could occur when processing an in-flight message after transaction has been failed over and committed.